Packs/CommonPlaybooks/Playbooks/playbook-Threat_Hunting_-_Generic_6_5.yml

```diff
@@ -54,7 +54,7 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "4"
+      - "7"
     scriptarguments:
       IPAddress:
         simple: ${inputs.IPAddress}
```

Comment:
lets change all the "simple" inputs to complex and add unique transformer

Reply:
done